abstract/myst.yml

```
@@ -44,10 +35,10 @@ project:
 site:
   template: article-theme
   actions:
-    # - title: pdf
-    #   url: ARTICLE-NAME.pdf
-    #   static: true
-    # - title: slides
-    #   url: presentation/SLIDES-NAME.pdf
-    #   static: true
+    - title: poster
+      url: abstract/presentation/geophysics-in-a-changing-climate-eposter.pdf
+      static: true
+    - title: slides
+      url: abstract/presentation/geophysics-in-a-changing-climate.pdf
+      static: true
```

Here, the path should be relative to where the myst file is. Since we are already in the `abstract` folder, it doesn't need to be included in the path, we can just state that it is in the `presentation` folder
```suggestion
    - title: slides
      url: presentation/geophysics-in-a-changing-climate.pdf
      static: true
```